src/components/screens/management-screen.tsx

```diff
@@ -84,6 +91,46 @@ const ManagementScreen: React.FC = () => {
     setIsLoading(false);
   };
 
+  /**
+   * Event handler for billing rate update button click
+   */
+  const handleBillingRateUpdateClick = async () => {
+    if (!selectedPersonWithTotalTime || !selectedPersonWithTotalTime.personTotalTime) {
```

Comment:
These can be combined:
```suggestion
    if (!selectedPersonWithTotalTime?.personTotalTime) {
```